portfolio/src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.auto.value.AutoValue;
+
+/** Stores user login data. */
+@AutoValue 
+public abstract class User {
+  public abstract boolean loggedIn();
+  public abstract String loginURL();
```

Comment:
Something to consider as a cleanup. You could have the loginUrl and logoutUrl be Optional. Therefore instead of passing an empty string you make it known that it is not a required field. Optional is better because it is more explicit.

Reply:
done